### PR TITLE
card html

### PR DESCRIPTION
**Pull request by b0llu**
Added Card to Documentation site
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
Looks great to be merged

File: Components/Card/card.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        They should be easy to scan for relevant and actionable information. Elements, like text and images, should be
+        placed on them in a way that clearly indicates hierarchy.
+      </p>
+      <div class="divider"></div>
+      <h1 class="content-title">Card with Badge</h1>
+      <p class="sub-content">
+        A badge on card is used to show something important about a product in single word or a icon.
+        <div class="content-container">
+          <div class="card-container">
+            <img src="../../Assets/card-image-1.jpg" alt="A graphic card image">
```
Comment (hsnice16) [inline Components/Card/card.html]: Nice work. Just one suggestion, I think, you can use `loading=lazy` with the `` to improve the performance.
Everything else looks good.

**Review by hsnice16 — APPROVED** [review]

File: Components/Card/card.html (hunk 1 above)
Comment (b0llu) [reply Components/Card/card.html]: Thanks for the suggestion. i will definitely add it!

**Review by partha8 — APPROVED** [review]
Looks good

**Pull request merged**